### PR TITLE
FT: Add changes for replication with GCP

### PR DESCRIPTION
**Pull request by bennettbuchanan**

### CODE REVIEW

File: constants.js [hunk 1]
```diff
@@ -115,6 +115,7 @@ const constants = {
     legacyLocations: ['sproxyd', 'legacy'],
     /* eslint-disable camelcase */
     externalBackends: { aws_s3: true, azure: true, gcp: true },
+    replicationBackends: ['aws_s3', 'azure', 'gcp'],
```
Comment (ploki) [inline constants.js]: I might prefer the same construct as for the ```externalBackends``` constant

Comment (bennettbuchanan) [reply constants.js]: Good call. Updated!

**Comment by bennettbuchanan** [comment]
@ironman-machine r+

**Comment by bennettbuchanan** [comment]
@ironman-machine r-

**Comment by bennettbuchanan** [comment]
@ironman-machine 
RUN_BACKBEAT_CRR_TESTS=1
r+

**Comment by bennettbuchanan** [comment]
@ironman-machine 
RUN_BACKBEAT_CRR_TESTS=1
try

**Comment by bennettbuchanan** [comment]
@ironman-machine
RUN_BACKBEAT_CRR_TESTS=1
try

**Pull request merged**